src/components/ErrorModal/ErrorModal.js

```diff
@@ -35,7 +35,7 @@ const ErrorModal = () => {
 
   useEffect(() => {
     const onError = error => {
-      error = error.detail || error.message;
+      error = error.detail.message || error.detail || error.message;
```

Comment:
Will `error.detail` be `undefined` or `null`? I guess it will be an empty object at least since it's from a DOM CustomEvent?

Reply:
Hm, it is null by default but I believe we are sending a message most of the time. I will add a better check just in case.